fdbbackup/backup.actor.cpp

```diff
@@ -77,22 +77,22 @@ using std::endl;
 enum class ProgramExe { AGENT, BACKUP, RESTORE, FASTRESTORE_TOOL, DR_AGENT, DB_BACKUP, UNDEFINED };
 
 enum class BackupType {
-	UNDEFINED = 0,
-	START,
-	MODIFY,
-	STATUS,
-	ABORT,
-	WAIT,
-	DISCONTINUE,
-	PAUSE,
-	RESUME,
-	EXPIRE,
-	DELETE,
```

Comment:
My bet is it's `DELETE`: https://github.com/wine-mirror/wine/blob/47ac628b4a4e476c1b044765c95d5be2a7101d14/include/winnt.h#L5378

Reply:
Thanks, that was the issue.